File: Android 10 - Android services, Work managers - executing long running tasks/ServiceExercise_2/app/src/main/java/com/programming/android/sdu/serviceexercise/JokeWebService.java
package com.programming.android.sdu.serviceexercise;

import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Created by slapocolypse on 3/12/18.
 */

public interface JokeWebService {
    @GET("jokes/random")
    Call<Joke> randomJoke();
}

File: Android 10 - Android services, Work managers - executing long running tasks/ServiceExercise_2/app/src/main/java/com/programming/android/sdu/serviceexercise/JokeWebService.kt
package com.programming.android.sdu.serviceexercise

import retrofit2.Call
import retrofit2.http.GET

/**
 * Created by slapocolypse on 3/12/18.
 */
interface JokeWebService {
    @GET("jokes/random")
    fun randomJoke(): Call<Joke?>?
}